payments-core/src/main/java/com/stripe/android/view/AuthActivityStarterHost.kt

```diff
@@ -0,0 +1,74 @@
+package com.stripe.android.view
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.fragment.app.Fragment
+
+/**
+ * A representation of an object (i.e. Activity or Fragment) that can start an activity.
+ */
+@Suppress("DEPRECATION")
```

Comment:
nit: I think moving this to the actual location where the deprecated methods is called makes it more readable